great-idea-website/css/index.css

```diff
@@ -63,4 +63,124 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+p {
+	font-family:'Montserrat', sans-serif;
+}
+
+nav {
+	margin-top: 30px;
+	margin-bottom: 30px;
+	position: relative;
+}
+
+nav a {
+	color: grey;
+	text-decoration: none;
+	margin-left: 100px;
+}
+
+nav img {
+	margin-left: 100px;
+	position: absolute;
+	top: -12px;
+}
+
+h1 {
+	font-size: 80px;
+}
+
+.button {
+	margin-left: 240px;
+	margin-bottom: 25px;
+}
+
+header {
+	padding-top: 20px;
+	padding-bottom: 20px;
+}
+
+header p {
+	border: 1px solid black;
+	padding-top: 5px;
+	padding-bottom: 5px;
+	width: 175%;
+	text-align: center;
+
+}
+
+.container {
+	border-bottom: 1px solid black;
+	text-align: center;
+
```

Comment:
i would add a width here and also a margin 0 auto`.container {
	border-bottom: 1px solid black;
	text-align: center;
 `